src/components/EmojiSuggestions/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
Please add a brief description to the top of the file to explain why there are platform specific files and what the differences are.

Reply:
Yeah, I think the file pattern is fine here, but I think we need a comment somewhere in the code which explains why this is being done. I've covered most of that in the other thread, so I will resolve this one.